packages/nimble-components/src/table-column/date-text/tests/table-column-date-text.mdx

```diff
@@ -16,7 +16,11 @@ The <Tag name={tableColumnDateTextTag}/> column is used to display date-time fie
 
```

Comment:
The docs for the date and duration columns are showing errors for me in the latest storybook build.

![image](https://github.com/ni/nimble/assets/10500124/eb84581c-cad5-419b-a571-2a1ecbe4ea01)

https://60e89457a987cf003efc0a5b-wlnihdzmym.chromatic.com/?path=/docs/components-table-column-duration-text--docs
https://60e89457a987cf003efc0a5b-wlnihdzmym.chromatic.com/?path=/docs/components-table-column-date-text--docs

Reply:
It should be fixed now. I also ended up moving the location of the shared docs for not mixing `undefined` and `null` so that it renders correctly in the docs.